mne/decoding/ems.py

```diff
@@ -68,7 +68,8 @@ def compute_ems(epochs, conditions=None, picks=None, verbose=None, n_jobs=1):
     else:
         epochs = epochs[conditions]
 
-    epochs.drop_bad_epochs()
+    if epochs._bad_dropped is False:
```

Comment:
woops, I wasn't sure if I was doing the right thing. I knew it wasn't good to expose them to the users but wasn't sure if it was ok to use them in MNE methods. Now I know :)